lib/src/vector_drawable.dart

```diff
@@ -859,6 +865,9 @@ class DrawableRoot implements DrawableParent {
   @override
   final List<Drawable> children;
 
+  @override
+  String get id => 'root';
```

Comment:
I think we should avoid this - someone might use that string in an SVG, and we should just take the `@id` attribute from the `<svg>` element itself instead.

Reply:
Yes, this has been fixed on new commit.
I also have added test case for this.
Thank you